sigle/src/utils/regex.ts

```diff
@@ -1 +1,9 @@
 export const hexRegex = /^#([A-Fa-f0-9]{6})$/;
+
+export const isValidEmail = (email: string) => {
+  return email
```

Comment:
is there a way to get the regex from next-auth somehow?

Reply:
had a look and doesn't look like there is a utility from the library that we can use to validate emails, but some alternative approaches mentioned here https://github.com/nextauthjs/next-auth/issues/1554